app/supportAssignment/supportAssignmentApp.js

```diff
@@ -0,0 +1,39 @@
+window.supportAssignmentApp = angular.module("supportAssignmentApp", ["sharedApp", "ngRoute"]);
+
+supportAssignmentApp.config(function ($routeProvider) {
+	return $routeProvider
+	.when("/:workgroupId/:year/:termShortCode", {
```

Comment:
Move the .when up to the same line as 'return'

Reply:
I think the intent of this format was to visually connect the .when() with the .otherwise() a few lines down. In routers where there are multiple whens its especially helpful
Example:
https://github.com/ucdavis/ipa-client-angular/blob/master/app/summary/summaryApp.js